lib/standalone-single-spa.js

```diff
@@ -58,98 +58,108 @@ class StandaloneSingleSpaPlugin {
-              tagName: "main",
-              voidTag: false,
-              innerHTML: this.getDisabledHtml({ mainScriptSrc }),
-            });
+    if (this.options.disabled) {
+      // don't load the scripts - just show the disabled html
+      scripts.splice(0, scripts.length);
 
-            return;
-          }
+      scripts.push({
+        tagName: "main",
+        voidTag: false,
+        innerHTML: this.getDisabledHtml({ mainScriptSrc }),
+      });
 
-          // Delete the main script, since it will be loaded later by SystemJS
-          data.assetTags.scripts.splice(0, 1);
+      return;
+    }
 
-          const importMap = {
-            imports: {
-              "single-spa":
-                "https://cdn.jsdelivr.net/npm/single-spa/lib/system/single-spa.dev.js",
-              [this.options.appOrParcelName]: mainScriptSrc,
-            },
-          };
+    // Delete the main script, since it will be loaded later by SystemJS
+    scripts.splice(0, 1);
 
-          for (let name in this.options.importMap.imports) {
-            importMap.imports[name] = this.options.importMap.imports[name];
-          }
+    const importMap = {
+      imports: {
+        "single-spa":
+          "https://cdn.jsdelivr.net/npm/single-spa/lib/system/single-spa.dev.js",
+        [this.options.appOrParcelName]: mainScriptSrc,
+      },
+    };
 
-          const pluginRuntime = this.options.isParcel
-            ? this.parcelRuntime()
-            : this.applicationRuntime();
+    for (let name in this.options.importMap.imports) {
+      importMap.imports[name] = this.options.importMap.imports[name];
+    }
 
-          if (this.options.importMapUrl) {
-            data.assetTags.scripts.push({
-              tagName: "script",
-              voidTag: false,
-              attributes: {
-                type: "systemjs-importmap",
-                src: this.options.importMapUrl.href,
-              },
-            });
-          }
-          data.assetTags.scripts.push({
-            tagName: "script",
-            voidTag: false,
-            attributes: { type: "systemjs-importmap" },
-            innerHTML: JSON.stringify(importMap, null, 2),
-          });
-          data.assetTags.scripts.push({
-            tagName: "script",
-            voidTag: false,
-            attributes: {
-              defer: false,
-              src: "https://cdn.jsdelivr.net/npm/systemjs/dist/system.js",
-            },
-          });
-          data.assetTags.scripts.push({
-            tagName: "script",
-            voidTag: false,
-            attributes: {
-              defer: false,
-              src: "https://cdn.jsdelivr.net/npm/systemjs/dist/extras/amd.js",
-            },
-          });
-          data.assetTags.scripts.push({
-            tagName: "script",
-            voidTag: false,
-            attributes: {
-              defer: false,
-              src:
-                "https://cdn.jsdelivr.net/npm/systemjs/dist/extras/named-exports.js",
```

Comment:
The named-exports systemjs extra is no longer needed for new versions of systemjs, so I deleted this.